src/utils/time.ts

```diff
@@ -247,6 +247,10 @@ export const normalizeTimestampToMilliseconds = (timestamp: number) => {
     : timestamp * ONE_SECOND_IN_MILLISECONDS;
 };
 
+export const getWeekday = (date: Date | number) => {
+  return format(date, "EEEE");
+};
+
```

Comment:
should be deleted?

Reply:
no.

Reply:
I’m kind of on @mike-lvov’s side with this one as well. Why does this need to stay? Seems wrong to me.